lib/spack/docs/packaging_guide.rst

```diff
@@ -3595,7 +3776,7 @@ Shell command functions
 Recall the install method from ``libelf``:
 
 .. literalinclude::  _spack_root/var/spack/repos/builtin/packages/libelf/package.py
-   :pyobject: Libelf.install
+   :pyobject: AutotoolsBuilder.install
```

Comment:
I don't really see any reason to modify libelf in this PR, it only supports a single build system

Reply:
It's a leftover from previous changes. I'll fix that.

Many goals were moving targets during the writing of this PR. The entire PR started with the intent that in the long term we would move to the new format only. The doc was written back then. Later we decided to postpone this decision to a later point and I think the sentiment is to keep the current recipes as they are forever, if they don't need multiple build systems.

Needless to say the bad side of this is that we need the adapter code to be around forever :grimacing: